bin.js

```diff
@@ -45,13 +50,21 @@ function getConfig(file) {
     // point to the particular config
     configFile: file,
   })
-  var config = cliEngine.getConfigForFile(file)
-  return config
+  return cliEngine.getConfigForFile(file)
+}
+
+function getCurrentRules(conf) {
+  var rules = Object.keys(conf.rules)
+  return rules
 }
 
-function getRules() {
-  var configFile = getConfigFile()
-  var config = getConfig(configFile)
-  var rules = Object.keys(config.rules)
+function getPluginRules(conf) {
+  var rules = []
+  var plugins = conf.plugins
+  if (plugins) {
+    plugins.forEach(function normalizePluginRule(plugin) {
+      rules.concat(require('eslint-plugin-' + plugin).rules)
```

Comment:
Another thing, the rule name should be prepended with the plugin name. e.g. rule `display-name` for `eslint-plugin-react` should be like `react/display-name`